Create lesson: sphere-area (#199)
3b1b/3Blue1Brown.com: public/content/lessons/2018/sphere-area/index.mdx

This lends itself to a rather bizarre way to animate the relation, where instead of projecting each rectangular piece, you rotate each one and rearrange them to make the cylinder.

<Figure video="rotation-animation.mp4" />

</Accordion>

Now, if you’re really thinking critically, you might still not be satisfied that this shows that the surface area of the sphere equals the area of this cylinder label. After all, these little rectangles only *approximate* the relevant areas.

Well, the idea is that this approximation gets closer and closer to the true value for finer and finer coverings. Since for any specific covering, the sphere rectangles have the same area as the cylinder rectangles, whatever values each of these two series of approximations are approaching must actually be the same.

<Figure image="equal-in-limit.png" />

> I mean, as you get really aggressively philosophical about what we even mean by surface area, these sorts of rectangular approximations and not just aids in our problem-solving toolbox, they end up serving as a way of rigorously defining the area of smooth curved surfaces.

Reviewer: ```suggestion
As you get more philosophical and ask what exactly we mean by surface area, these sorts of rectangular approximations and not just aids in our problem-solving toolbox, they end up serving as a way of rigorously defining the area of smooth curved surfaces, though often [some care is required](https://en.wikipedia.org/wiki/Schwarz_lantern).
```

<Figure image="surface-area-definition.png" />

This kind of reasoning is essentially calculus, just stated without any of the jargon. In fact, I think neat geometric arguments like this, which require no background in calculus to understand, can serve as a great way to tee things up for new calculus students so that they have the core ideas before seeing the definitions which make them precise (rather than the other way around).

<Figure
  image="preparing-for-calculus.png"
  caption="These geometric arguments are great preparation for understanding calculus."
/>

### Connection to circles

> So as I said before, if you’re itching to see a direct connection to four circles, one nice way is to unwrap these circles into triangles. If this is something you haven’t seen before, I go into much more detail about why this works in <LessonLink id="essence-of-calculus">the first lesson of the calculus series</LessonLink>.

Reviewer: ```suggestion
If you’re itching to see a direct connection to four circles, one nice way is to unwrap these circles into triangles. If this is something you haven’t seen before, I go into much more detail about why this works in <LessonLink id="essence-of-calculus">the first lesson of the calculus series</LessonLink>.
```

The basic idea is to relate thin concentric rings of the circle with horizontal slices of this triangle.

<Figure
  image="ring-to-slice.png"
  caption="Each circle is unwrapped such that a thin ring of the circle corresponds to a thin line of the triangle."
/>

Because the circumference of each such ring increases linearly in proportion to the radius, always equal to $2\pi r$, when you unwrap them all and line them up, their ends will form a straight line (as opposed to some kind of curvey shape), giving you a triangle with a base of $2\pi r$ and a height of $r$.

And four of these unwrapped circles fit into our rectangle, which is in some sense an unwrapped version of the sphere’s surface.

<Figure image="triangles-fit-rectangle.png" />

## Second approach

> Nevertheless, you might wonder if there’s a way than this to relate the sphere directly to a circle with the same radius, rather than going through this intermediary of the cylinder.

Reviewer: ```suggestion
Nevertheless, you might wonder if there’s a way to relate the sphere directly to a circle with the same radius, rather than going through this intermediary of the cylinder.
```

<Figure image="direct-proof.png" />

> I do have a proof for you to this effect, leveraging a little trigonometry, though I have to admit I still think the comparison to the cylinder wins out on elegance.

Reviewer: ```suggestion
If you're willing to roll up your sleeves and leverage a little trigonometry, it is actually possible to draw this more direct connection.
```

> I’m a big believer that the best way to really learn math is to do problems yourself, which is a bit hypocritical coming from a channel essentially consisting of lectures. So I’m going to try something a little different here and present the proof as a heavily guided sequence of exercises. Yes, I know that’s less fun and it means you have to pull out some paper to do some work, but I guarantee you’ll get more out of it this way.

Reviewer: ```suggestion
I’m a big believer that the best way to really learn math is to do problems yourself, which is a bit hypocritical coming from a channel essentially consisting of lectures. So let's try something a little different here and present the proof as a heavily guided sequence of exercises. Yes, I know that’s less fun and it means you have to pull out some paper to do some work, but I guarantee you’ll get more out of it this way.
```

<FreeResponse>

Each shadow corresponds to one of the alternating rings of the sphere. The special property of this correspondence, spelled out in question 3, is that corresponding ring has an area double that of the shadow.

This means that the alternating rings have a total area that is exactly double the total area of the shadows.

</FreeResponse>

### Question 5
Bring it on home. Why does this imply that the area of the circle is exactly ¼ the surface area of the sphere, particularly as we consider thinner and thinner rings?

<Figure image="question-5.png" />

<FreeResponse>

All together, the top-hemisphere ring shadows form a perfect circle of radius $R$ on the xy-plane. This means that the sum of all their areas is exactly $\pi R^2$.

<Figure image="shadows-form-circle.png" video="shadows-form-circle.mp4" />

And because of our special correspondence, we know that the total area of all the alternating rings on the sphere is exactly double the total shadow area. So adding up the areas of these rings of the sphere gives a surface area of $2\pi R^2$:

<Figure
  image="alternating-rings.png"
  caption="The total surface area of these rings is $2\pi R^2."
/>

> To get the full sphere surface area for *all* the rings (not just every-other ring), we double our result for an area of $4\pi R^2$.

Reviewer: ```suggestion
In the limit as these rings get ever thinner, the area of the alternating rings approaches half the total surface area. This is because each ring has approximately the same area as its immediate neighbors in that limiting case, so for example the sum of the areas of all the odd-numbered rings will equal the sum of the areas of the even-numbered rings. So to get the full surface area of the sphere, we double the total area of the alternating rings, which is $4\pi R^2$.
```

</FreeResponse>

## The more general shadow formula
And finally, I’d be remiss not to make a brief mention of the fact that the surface area of a sphere is a specific instance of a much more general fact: If you take any convex shape, and look at the average area of all its shadows, averaged over all possible orientations in 3D space, the surface area of the solid will be precisely 4 times that average shadow area.

<Figure image="average-shadow.png" video="average-shadow.mp4" show="video" />

As to why this is true? I’ll leave those details for another day.

> [^1]: Maybe you've seen this as $\sin(\theta) = \frac{\text{opposite}}{\text{hypotenuse}}$.

Reviewer: ```suggestion
[^1]: $\sin(\theta) = \frac{\text{opposite}}{\text{hypotenuse}}$.
```
